target/linux/ipq806x/image/Makefile

```diff
@@ -81,6 +81,21 @@ define Device/ZyXELImage
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to $$$$(BLOCKSIZE) | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
+define Device/askey_rt4230w-rev6
+	$(call Device/LegacyImage)
+	DEVICE_VENDOR := Askey
+	DEVICE_MODEL := RT4230W
+	DEVICE_VARIANT := REV6
+	SOC := qcom-ipq8065
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	DEVICE_PACKAGES := ath10k-firmware-qca9984-ct
+	KERNEL_IN_UBI := 1
+	IMAGES := nand-sysupgrade.bin
```

Comment:
Where is the factory image mentioned in the commit message?

Reply:
I switched to using sysupgrade and forgot to change that

Reply:
So you don't use factory image anymore during installation. But still nand-factory ubi image might be useful for someone else (unless the image is totally absurd for this device). I guess you can just use `$(call Device/UbiFit)` here.